Indicate that abs() of the minimum integer value for signed dtypes is undefined (#250)
data-apis/array-api: spec/API_specification/elementwise_functions.md

(function-abs)=
### abs(x, /)

> Calculates the absolute value for each element `x_i` of the input array `x` (i.e., the element-wise result has the same magnitude as the respective element in `x` but has positive sign). For signed integer data types, the `abs` of smallest representable integer is implementation-defined.

Reviewer: Should we move this to the special cases section? Or maybe just move to a note?

Calculates the absolute value for each element `x_i` of the input array `x` (i.e., the element-wise result has the same magnitude as the respective element in `x` but has positive sign).

```{note}

> For signed integer data types, the absolute value of minimum representable integer is implementation-dependent.

Reviewer: ```suggestion
For signed integer data types, the absolute value of the minimum representable integer is implementation-dependent.
```

Calculates the absolute value for each element `x_i` of the input array `x` (i.e., the element-wise result has the same magnitude as the respective element in `x` but has positive sign).

```{note}

> 

Reviewer: ```suggestion
```

> For signed integer data types, the `abs` of smallest representable integer is implementation-dependent.

Reviewer: ```suggestion
For signed integer data types, the absolute value of smallest representable integer is implementation-dependent.
```

> For signed integer data types, the absolute value of smallest representable integer is implementation-dependent.

Reviewer: ```suggestion
For signed integer data types, the absolute value of minimum representable integer is implementation-dependent.
```
